object-search/app.py

```diff
@@ -72,51 +70,35 @@ def dryrun():
-)
-@click.option('--data_path', '-p', default=data_path)
-@click.option('--num_docs', '-n', default=num_docs)
-@click.option('--batch_size', '-b', default=batch_size)
-@click.option('--overwrite_workspace', '-overwrite', default=True)
-def main(task, return_image, data_path, num_docs, batch_size, overwrite_workspace):
+    '--return_image',
+    '-r',
+    default='original',
+    type=click.Choice(['original', 'object'], case_sensitive=False))
+@click.option('--num_docs', '-n', default=MAX_DOCS)
+def main(task: str, return_image: str, num_docs: int):
     config()
     workspace = os.environ['WORKDIR']
     if 'index' in task:
         if os.path.exists(workspace):
-            if overwrite_workspace:
-                clean_workdir()
-            else:
-                print(
-                    f'\n +------------------------------------------------------------------------------------+ \
-                        \n |                                   🤖🤖🤖                                           | \
-                        \n | The directory {workspace} already exists. Please remove it before indexing again.  | \
-                        \n |                                   🤖🤖🤖                                           | \
-                        \n +------------------------------------------------------------------------------------+'
-                )
-                sys.exit(1)
-
-    elif 'query' in task:
-        if not os.path.exists(workspace):
-            print(f"The directory {workspace} does not exist. Please index first via `python app.py -t index`")
-
+            logger.error(f'\n +---------------------------------------------------------------------------------+ \
+                    \n |                                   🤖🤖🤖                                        | \
+                    \n | The directory {workspace} already exists. Please remove it before indexing again. | \
+                    \n |                                   🤖🤖🤖                                        | \
+                    \n +---------------------------------------------------------------------------------+')
+            sys.exit(1)
     if task == 'index':
-        f = Flow.load_config('flow-index.yml')
-        with f:
-            f.index_files(data_path, batch_size=batch_size, read_mode='rb', size=num_docs)
-    elif task == 'index_restful':
+        index(num_docs)
+    if task == 'index-restful':
         index_restful(num_docs)
-    elif task == 'query':
-        f = Flow.load_config(f'flow-query-{return_image}.yml')
-        with f:
-            f.block()
-    elif task == 'query_restful':
-        query_restful(return_image)
-    elif task == 'dryrun':
+    if task == 'query-restful':
+        if not os.path.exists(workspace):
+            logger.error(f"The directory {workspace} does not exist. Please index first via `python app.py -t index`")
```

Comment:
I know that the string was there before, but if you touch the line anyway, can you please use single quote strings as per our standard?

Reply:
Sure, I changed it in the file: https://github.com/jina-ai/examples/pull/477/commits/fcfb02eccad7272d342a11e50412fe9cf1cf161d

What do you mean with touch the line away?😀